apps/website/src/components/shared/form/NumberField.tsx

```diff
@@ -1,21 +1,49 @@
 import { useCallback, useMemo, useRef, useState, type ReactNode } from "react";
-import type { AriaNumberFieldProps } from "react-aria";
-import { useLocale, useNumberField } from "react-aria";
-
+import {
+  type AriaNumberFieldProps,
+  type AriaButtonOptions,
+  useLocale,
+  useNumberField,
+  useButton,
+} from "react-aria";
 import { useNumberFieldState } from "react-stately";
+
 import { classes } from "@/utils/classes";
 
 import IconX from "@/icons/IconX";
 
+function NumberButton(
+  props: AriaButtonOptions<"button"> & { children: ReactNode },
+) {
+  const ref = useRef(null);
+  const { buttonProps } = useButton(props, ref);
+  return (
+    <button
+      className={classes(
+        "m-1 block w-12 rounded-full bg-gray-700 text-lg text-white",
```

Comment:
You might want to set a fixed height on these as well? They're not quite circles currently (and it was visually upsetting to me lol):

![image](https://github.com/alveusgg/alveusgg/assets/12371363/95f4814c-10a7-4e7b-af64-31fcb4d5dbbd)

I'm not sure w-12 is doing anything here currently as the parent container this is used in has a flex w/ width set to min-content and this doesn't have a flex-shrink 0, so it's being shrunk.

I got this looking nice by setting w-8 h-8 flex-shrink-0 (potentially also flex items-center justify-center)

```suggestion
        "m-1 block w-8 h-8 flex-shrink-0 flex items-center justify-center rounded-full bg-gray-700 text-lg text-white",
```